Modules/spwdmodule.c

```diff
@@ -256,5 +256,12 @@ static struct PyModuleDef spwdmodule = {
 PyMODINIT_FUNC
 PyInit_spwd(void)
 {
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
+                     "'nis' is deprecated and slated for removal in "
+                     "Python 3.13",
+                     7)) {
```

Comment:
Why do you use stack level 7 ?

Reply:
It seems to be what it take for the traceback to show the import statement and not some internal detail of importlib.